File: backend-quarkus/src/main/java/app/nanotes/backend/web/CookieNames.java
package app.nanotes.backend.web;

/**
 * Cookie and header names are part of the cross-implementation contract
 * (ADR 0005) — every backend must use these exact names so a frontend
 * implementation never needs backend-specific logic.
 */
public final class CookieNames {
    public static final String SESSION = "session";
    public static final String CSRF = "csrf_token";
    public static final String CSRF_HEADER = "X-CSRF-Token";

    private CookieNames() {}
}

File: backend-quarkus/src/main/kotlin/app/nanotes/backend/web/CookieNames.kt
package app.nanotes.backend.web

/**
 * Cookie and header names are part of the cross-implementation contract
 * (ADR 0005) — every backend must use these exact names so a frontend
 * implementation never needs backend-specific logic.
 */
object CookieNames {
    const val SESSION = "session"
    const val CSRF = "csrf_token"
    const val CSRF_HEADER = "X-CSRF-Token"
}
